hlint-flow.yaml

```diff
@@ -0,0 +1,36 @@
+- hint: 
+    lhs: 'f $! x'
+    note: 'Use `<!`'
+    rhs: 'f <! x'
+
+- hint: 
+    lhs: 'f $ x'
+    note: 'Use `<|`'
+    rhs: 'f <| x'
+
+- hint: 
+    lhs: 'x & f'
+    note: 'Use `|>`'
+    rhs: 'f |> x'
+
+- hint: 
+    lhs: 'f . g'
+    note: 'Use `<.`'
+    rhs: 'f <. g'
+
+
+- hint: 
+    lhs: 'f <| x'
+    note: 'Use `|>` for natural reading direction' 
+    rhs: 'x |> f'
```

Comment:
I typically use `<|` for passing a big `do` block to a function. Does this hint fire for code like this?

``` hs
liftIO <| do
  now <- getCurrentTime
  print now
```

Reply:
This has been bothering me also as I don't know how to do it. Perhaps this hint should be disabled?